### DISCUSSION TITLE
Rewrite yarpview and yarpscope without QML

### DISCUSSION THREAD

**Discussion by drdanz** [robotology/yarp#2077] [Ideas]
We never actually used any of the features offered by qml, but we had lots and lots of issues and bug reports due to the qml stuff.

Most of the issues regard:
* Missing menus
* Failed to find the qmldir or the qml plugin dll
* Filed to find runtime required plugins
* Issues with opengl

I think it's time that we give up on QML, and make them normal guis.

Opinions?

**Comment by Nicogene** [comment]
It will solve probably the problems with the WSL and various Xservers(#2071)

I am not aware of the advantages using qml, but if we are not exploiting it, I agree to convert them and get rid of the qml-related troubles.

**Comment by pattacini** [comment]
Agree on this 👍

**Comment by traversaro** [comment]
This would also solve indirectly https://github.com/robotology/robotology-superbuild/issues/134 ?
